test/packages/other/hit_count_assertion/manifest.yml

```diff
@@ -0,0 +1,24 @@
+format_version: 1.0.0
+name: hit_count_assertion
+title: Hit Count Assertion test
+# version is set to something very large to so this test package can
+# be installed in the package registry regardless of the version of
+# the actual apache package in the registry at any given time.
+version: 999.999.999
+description: Test for Hit Count Assertion
+categories: ["network"]
+release: experimental
```

Comment:
Tests are failing in CI because the package is not available for the version of the stack used. Experimental packages are not returned by the package registry by default, try removing this line.
```suggestion
```